src/main/java/in/co/gorest/api/rest/EndpointTypes.java

```diff
@@ -0,0 +1,67 @@
+package in.co.gorest.api.rest;
+
+import in.co.gorest.constans.IConstants;
+import org.apache.commons.lang3.tuple.Pair;
+
+public enum EndpointTypes implements IConstants {
```

Comment:
enum names should be singular, please rename all enums
```suggestion
public enum EndpointType implements IConstants {
```